src/analog.cpp

```diff
@@ -0,0 +1,44 @@
+#include "analog.h"
+#include "STM32TimerInterrupt_Generic.h"
```

Comment:
I moved `#include "STM32TimerInterrupt_Generic.h"` in here so that the code would compile, but after we rename these analog files it should be good to go back into the `.h` file.